features/step_definitions/assertion_steps.rb

```diff
@@ -299,12 +301,12 @@ def current_path_without_locale(path)
 end
 
 
-# Tests that an input or button with the given label is disabled.
-Then /^the "([^\"]*)" field should( not)? be set to "([^\"]*)"$/ do |label, negate, text_value|
+# Tests that an input has a given value
+Then /^the t\("([^"]*)"\) field should be set to "([^\"]*)"$/ do |i18n_key, text_value|
```

Comment:
I think the step method defined at line 425 in this file would also work.   However, you may prefer the wording of this step over that other one.